.github/workflows/production-dependencies-license-check.yml

```diff
@@ -51,7 +51,7 @@ jobs:
       - name: Run license-checker
         id: licenseChecker
         if: ${{ steps.dependencies_have_changed.outputs.dependencies-have-changed == 'true' }}
-        run: echo DEPENDENCIES=$(npx license-checker-rseidelsohn@4 --direct --production --onlyAllow="${{ steps.allowed-licenses.outputs.FILE }}" --json --excludePrivatePackages) >> $GITHUB_OUTPUT
+        run: echo DEPENDENCIES=$(npx license-checker-rseidelsohn@4.2.9 --direct --production --onlyAllow="${{ steps.allowed-licenses.outputs.FILE }}" --json --excludePrivatePackages) >> $GITHUB_OUTPUT
```

Comment:
question: didn't `@4` always take the latest on v4?

Reply:
Yes, but the latest version were broken :-(